mpqp/core/instruction/measurement/basis_measure.py

```diff
@@ -49,12 +50,14 @@ class BasisMeasure(Measure):
 
     def __init__(
         self,
-        targets: list[int],
+        targets: Optional[list[int]] = None,
```

Comment:
We didn't do the same for ExpectationMeaasure right ?

Reply:
Yes, only Basismeasure is allowed to have no target